From c8659d4d0459c51422754b677f92a5ad2063c043 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Thu, 14 Oct 2021 23:27:00 -0300
Subject: [PATCH 273/322] [FIO toimx] imx8mp_evk: don't forcely enable USB

Prefer having the USB related configs to be enabled and supported via
u-boot config instead.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 include/configs/imx8mp_evk.h | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/include/configs/imx8mp_evk.h b/include/configs/imx8mp_evk.h
index a298e8995d1..c29d4af1885 100644
--- a/include/configs/imx8mp_evk.h
+++ b/include/configs/imx8mp_evk.h
@@ -266,14 +266,6 @@
 
 #define CONFIG_SYS_I2C_SPEED		100000
 
-/* USB configs */
-#ifndef CONFIG_SPL_BUILD
-
-#define CONFIG_CMD_USB_MASS_STORAGE
-#define CONFIG_USB_GADGET_MASS_STORAGE
-#define CONFIG_USB_FUNCTION_MASS_STORAGE
-#endif
-
 #define CONFIG_USB_MAX_CONTROLLER_COUNT         2
 #define CONFIG_USBD_HS
 #define CONFIG_USB_GADGET_VBUS_DRAW 2
-- 
2.34.1

